t/plugin/proxy-rewrite.t

```diff
@@ -403,3 +403,53 @@ host: localhost
 x-real-ip: 127.0.0.1
 --- no_error_log
 [error]
+
+
+
+=== TEST 14: set route(rewrite uri args)
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "plugins": {
+                            "proxy-rewrite": {
+                                "uri": "/plugin_proxy_rewrite_args",
```

Comment:
do we need to support `/plugin_proxy_rewrite_args?foo=123` in uri?

Reply:
not supported.

Reply:
I want to look at the feedback in the community first to determine if it needs to be developed.